genesyscloud/provider/provider.go

```diff
@@ -268,14 +268,23 @@ func InitClientConfig(data *schema.ResourceData, version string, config *platfor
 		RetryWaitMax: time.Second * 30,
 		RetryMax:     20,
 		RequestLogHook: func(request *http.Request, count int) {
-			if count > 0 && request != nil {
-				log.Printf("Retry #%d for %s %s", count, request.Method, request.URL)
+			sdkDebugRequest := newSDKDebugRequest(request, count)
```

Comment:
So I restructured the logging in the core provider so it logs before and after each HTTP request.  After the AirBNB work, it did not make sense to just log information only when somethings fails.  These log messages only show up when the TF_LOG environment variable is set.